src/assets/Comment.jsx

```diff
@@ -1,15 +1,20 @@
 import React from "react"
 import { useState } from 'react'
-import Replies from "./Replies.jsx"
 import DeleteModal from "./DeleteModal"
 import NewComment from "./NewComment"
 import CommentCSS from "./comment.module.css"
 
 export default function Comment(props) {
     function handleAddClick() {
+        if (props.commentId) {

```

Comment:
You've used an "early return" pattern but functions should have a consistent return - if it returns something in some case, it should return something in other cases as well. If a function is "void" (doesn't really have a return value) then it's better to handle logic within "if / else" than to use "return" for early return pattern. So, in these cases I would say it's better and more readable to use "if / else" pattern without "return".

Reply:
updated both functions